db/schema.rb

```diff
@@ -99,10 +122,20 @@
     t.datetime "updated_at"
   end
 
+  create_table "suppliers", force: true do |t|
+    t.integer  "group_id"
+    t.integer  "producer_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "suppliers", ["group_id"], name: "index_suppliers_on_group_id", using: :btree
+  add_index "suppliers", ["producer_id"], name: "index_suppliers_on_producer_id", using: :btree
+
   create_table "users", force: true do |t|
     t.string   "email",           null: false
-    t.string   "first_name",      null: false
-    t.string   "last_name",       null: false
+    t.string   "first_name"
```

Comment:
I'm not sure, but if you think so let's go with required `first_name` and `last_name` attributes